src/components/FrontEndRolePage/__tests__/FrontEndRolePage.test.js

```diff
@@ -2,14 +2,20 @@ import React from 'react';
 import { create } from 'react-test-renderer';
 
 import FrontEndRolePage from '../FrontEndRolePage';
+import { MemoryRouter } from 'react-router-dom';
+import { FRONT_END_ROLE } from '../../../constants/roles';
 
 jest.mock('../../Characters', () => ({
```

Comment:
It would increase snapshot readability if we will mock more components here (check **BackEndRolePage.test.js** for reference).

Reply:
Fixed in c6073fe803add232415561bcb8753e478eb778bb